stack-easybib/recipes/role-publicapi.rb

```diff
@@ -4,8 +4,8 @@
 include_recipe 'php::module-zip'
 include_recipe 'php::module-zlib'
 
-include_recipe 'easybib-deploy::easybib'
-
+include_recipe 'stack-easybib::deploy-easybib'
+include_recipe 'stack-easybib::deploy-gearman-worker'
```

Comment:
This line is probably unneeded, gearman workers are handled in `stack-easybib::role-gearmanw`